griffin/secret_management_util.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+import hvac
+
+from griffin.griffin.decorators import singleton
+from .config import ENTITY_NAME, ENV, REGION, SERVICE_NAME, HASHICORP_URL, AWS_INSTANCE, VAULT_TOKEN, MOUNT_POINT, KEY
```

Comment:
Lets not use relative paths, use absolute paths please

Reply:
sure
have added changes, please check